generated/lspconfig-servers.json

```diff
@@ -0,0 +1,1705 @@
+[
+  {
+    "cmd": ["als"],
+    "desc": "https://github.com/agda/agda-language-server\n\nLanguage Server for Agda.\n",
```

Comment:
Was just curious because I see they basically always have the website and then a description in here. Made me think about how we provide a link to upstream docs for the plugins and was curious if we could grab the url and strip it out to be used for the proper link instead of crowding the description with it.

Reply:
I'd rather to do that in another PR, but it would be better I agree